pkg/webservice/handlers.go

```diff
@@ -236,3 +236,39 @@ func getNodeJSON(nodeInfo *cache.NodeInfo) *dao.NodeDAOInfo {
 		Schedulable: nodeInfo.IsSchedulable(),
 	}
 }
+
+func GetApplicationHistory(w http.ResponseWriter, r *http.Request) {
+	writeHeaders(w)
+
+	var result []*dao.ApplicationHistoryDAOInfo
+	records := imHistory.GetRecords()
+	for _, record := range records {
+		element := &dao.ApplicationHistoryDAOInfo{
+			Timestamp:         record.Timestamp.UnixNano(),
+			TotalApplications: strconv.Itoa(record.TotalApplications),
```

Comment:
in general, please check spacing. is it tab or spaces?
code allignment?

Reply:
What is the intended usage of the `go fmt` command?

I have the following error message running it from the project's root:
```
can't load package: package github.com/apache/incubator-yunikorn-core: unknown import path "github.com/apache/incubator-yunikorn-core": cannot find module providing package github.com/apache/incubator-yunikorn-core
```